src/sidebar.jsx

```diff
@@ -154,10 +156,10 @@ export const SidebarPanelDetails = ({
                                 {cockpit.format(
                                     cockpit.ngettext(
                                         "$0 item $1", "$0 items $1",
-                                        currentDirectory.items_cnt.all
+                                        item_count
                                     ),
-                                    currentDirectory.items_cnt.all,
-                                    cockpit.format("($0 hidden)", currentDirectory.items_cnt.hidden)
+                                    item_count,
+                                    showHidden ? "" : cockpit.format("($0 hidden)", hidden_item_count)
```

Comment:
This should be translated with ngettext.  "hidden" might be declined differently depending on the number.